### ISSUE TITLE
[WASM] ComboBox does not open the drop down when using spacebar

### ISSUE THREAD

**Issue by jsinsel** [unoplatform/uno#7759]
### Current behavior

Have a view with a combobox.  Navigate to the combobox with tab navigation.  When it is focused, use spacebar to open the drop down.  Does not work in WASM.

### Expected behavior

Spacebar will open the drop down.

### How to reproduce it (as minimally and precisely as possible)

Attaching a sample sln.
[App1.Shared.zip](https://github.com/unoplatform/uno/files/7823860/App1.Shared.zip)

### Works on UWP/WinUI

Yes

### NuGet package version(s)

```
<PackageReference Include="Uno.Material" Version="1.0.3" />
<PackageReference Include="Uno.Microsoft.Toolkit.Uwp.UI.Controls" Version="7.1.0-build.45.gd2e4192c8c" />
<PackageReference Include="Uno.Microsoft.Toolkit.Uwp.UI.Controls.DataGrid" Version="7.1.0-build.45.gd2e4192c8c" />
<PackageReference Include="Uno.Microsoft.Xaml.Behaviors.Uwp.Managed" Version="2.0.3-uno.95" />
<PackageReference Include="Uno.UI.WebAssembly" Version="3.11.6" />
<PackageReference Include="Uno.UI.RemoteControl" Version="3.11.6" Condition="'$(Configuration)'=='Debug'" />
<PackageReference Include="Uno.Wasm.Bootstrap" Version="3.0.0" />
<PackageReference Include="Uno.Wasm.Bootstrap.DevServer" Version="3.0.0" />
```

### Affected platforms

WebAssembly

### IDE

Visual Studio 2022

### IDE version

Version 17.1.0 Preview 1.0
